django/src/rdwatch/views/site_observation.py

```diff
@@ -167,16 +170,22 @@ def site_observations(request: HttpRequest, evaluation_id: UUID4):
     return queryset
 
 
+class GenerateImagesSchema(Schema):
+    constellation: Literal['WV', 'S2', 'L8'] = ('WV',)
+    dayRange: int = (14,)
+    noData: int = (50,)
+    overrideDates: None | list[str] = (None,)
+    force: bool = (False,)
+    scale: Literal['default', 'bits', 'custom'] = ('default',)
+    scaleNum: None | list[int] = (None,)
+    bboxScale: None | float = (1.2,)
```

Comment:
Why do these default to tuples? As they are, the default values don't adhere to the type hints.

```suggestion
    constellation: Literal['WV', 'S2', 'L8'] = 'WV'
    dayRange: int = 14
    noData: int = 50
    overrideDates: None | list[str] = None
    force: bool = False
    scale: Literal['default', 'bits', 'custom'] = 'default'
    scaleNum: None | list[int] = None
    bboxScale: None | float = 1.2
```

Reply:
I feel like that was done by a formatter (and not our standard on) not me for some reason.  It doesn't make sense that I would have done that.

Reply:
I don't see anything in the commit history, guess I was just having a weird moment.